ci-operator/step-registry/rosa/tests/runtime/day1-post/rosa-tests-runtime-day1-post-ref.yaml

```diff
@@ -18,7 +18,7 @@ ref:
     default: ""
     documentation: The profile for cluster provision. The profile specifies the configuration of the cluster.
   - name: COMPUTE_MACHINE_TYPE
-    default: ""
+    default: "m5.xlarge"
```

Comment:
Please reset default value

Reply:
@radtriste Removed the default value to empty string and just added in the file the env variable that was missing